src/main/java/edu/iastate/ece/sd/sdmay2126/runner/selenium/FBASeleniumInputProgrammer.java

```diff
@@ -107,6 +108,46 @@ public void programInputs(Job job, WebElement scopedFBACard) throws SeleniumIden
         }
         foundRMItem.get(0).click();
 
+        //setting reaction knockouts
+        System.out.println("Setting reaction knockouts");
+        //clear out old selections if any
+        WebElement reactionKnockoutArea = scopedFBACard
+                .findElement(By.cssSelector("div[data-parameter='reaction_ko_list']"));
+        WebElement selectedItemsRK = reactionKnockoutArea
+                .findElement(By.cssSelector("div[data-element='selected-items']"));
+        boolean removeElementsFromReactionKnockout = true;
+        while (removeElementsFromReactionKnockout) {
+            try {
+                WebElement alreadySelectedKO = selectedItemsRK
+                        .findElement(By.cssSelector("span[class='fa fa-minus-circle']"));
+                alreadySelectedKO.click();
+
+            } catch (Exception e) {
+                removeElementsFromReactionKnockout = false;
+
+            }
+        }
+        //put in new elements
+        WebElement reactionKOSearchBox = reactionKnockoutArea
+                .findElement(By.cssSelector("input[class='form-contol']"));
+        WebElement availableKOItems = reactionKnockoutArea
+                .findElement(By.cssSelector("div[data-element='available-items']"));
+        LinkedList<String> reactionKOS = params.getReactionKnockouts();
+        if (reactionKOS != null) {
+            for (String reactionKO : reactionKOS
+            ) {
+                reactionKOSearchBox.clear();
+                reactionKOSearchBox.sendKeys(reactionKO);
+                List<WebElement> foundKOItems = availableKOItems
+                        .findElements(By.cssSelector("span[class='kb-btn-icon']"));
+                if (foundKOItems.size() == 0) {
+                    System.out.println("Unable to find: " + reactionKO);
+                } else {
+                    foundKOItems.get(0).click();
```

Comment:
Would multiple results be considered undefined behavior? Should we eagerly fail if that occurs?

```
...
} else if (foundKOItems.size() == 0) {
    foundKOItems.get(0).click();
} else {
    throw new Exception("Too many results found for KO value.")
}
```